### DISCUSSION TITLE
How to download Order Invoices using Amazon SP API?

### DISCUSSION THREAD

**Discussion by srinivas6degree** [amzn/selling-partner-api-models#3722] [General]
Hello,
We are developing application to enable our sellers to use multiple marketplaces. 
We are able to use the SP API to get listings as well as orders. However we also wanted to download the order Invoice.

Is there a API to do that?

Thanks,
Srinivasa Pradeep

**Comment by egastech** [comment]
I'm trying to get invoice for a particular orderId, but as of my search on SP-API I did not find any. 
Did you see any API for getting invoices for a particular Order or filters in getInvoices api ?

**Comment by jawheral** [comment]
Hi,

Retrieving invoices depends on the marketplace, the seller settings and other variables. Without further information, it is not possible to answer the question? 
Could you please share more details on your particular use case?

Best,
Jawher
Selling Partner API Solutions Architect

**Comment by egastech** [comment]
So can you tell me what is the Invoice API actually is? is it not invoices
for the orders ?

2. if so, how to get invoice details for orders ?

**Reply by jawheral** [reply]
Hi @egastech,

The [Invoices API](https://developer-docs.amazon.com/sp-api/docs/invoices-api-v2024-06-19-reference) is only relevant to the Brazil marketplace. To answer the questions, please provide information on the marketplace(s).

**Comment by egastech** [comment]
I want to get details about seller's orders and their invoices of each
orders as well.
Im creating a custom report for orders where these are the attributes i
require:

Order Date Invoice Date Invoice ID Invoice Number Channel Order Id Channel
Sub Order Id Replacement Order ID (Orginal Order ID) Replacement Sub Order
ID Buyer Name Buyer Address1 Buyer Address2 Buyer City Buyer State Buyer
Pin Code Buyer Phone Buyer Email Billing Name Billing Address1 Billing
Address2 Billing City Billing State Billing Pincode Billing Phone Billing
Email Sla Date Product Sku Code Product Name Channel Listing ID/ ASIN / FSN
Listing Sku Code Channel Id Channel Name Category Name Selling Price Per
Item Shipping Charge Per Item Qty Currency Short Code Order Type Promo
Discount Gift Wrap Transaction Charges Total COD Collectible Gift Message
GST Number Shipment Tracker Shipping Company Delivery Date Shipment Date
Order Status Tax Rate Hsn Code Cgst Amount Sgst Amount Igst Amount Tcs
Amount Tax Due Tax Due Date Tax Refund Tax Refund Date Return Reason Return
Sub Reason Return Comments Return Date Return Delivered Date Return
Tracking Id Extra Info Return Good Qty Return Bad Qty ExtraInfo Listing Fee
Shipping Fee Octroi Fee TCS / TDS Payment Fees Commission Fees Commission
Fee Percent Reverse Shipping Fee Cancellation Fee Fee Discounts Pick And
Pack Fee Storage Fee Apology Fee Service Tax Total Channel Fees My Offer
Share Settlement Amount Payment Received Payment Reference Payment Date
Payment Delta Warehouse Name Claim Ref No Claim Date Claim Status
Submission Status
